test/FastTests/Client/QueriesWithCustomFunctions.cs

```diff
@@ -2090,6 +2090,41 @@ select new
         }
 
         [Fact]
+        public void Query_With_Load_Where_Id_Is_Not_A_Path()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new User { Name = "Jerry", LastName = "Garcia", FriendId = "users/2"}, "users/1");
+                    session.Store(new Detail { Number = 15 }, "details/1");
+                    session.SaveChanges();
+                }
+
+                using (var session = store.OpenSession())
+                {
+                    var query = from u in session.Query<User>()
+                                let detail = session.Load<Detail>("details/1")
+                                select new
+                                {
+                                    Name = u.Name,
+                                    Detail = detail
+                                };
+
+                    Assert.Equal("FROM Users as u LOAD \"details/1\" as detail " +
```

Comment:
While `load "static value"` is important and should be supported, in this case, we shouldn't be putting the raw value in the query, instead, this should be written as `load $p0` or something like that, and we need to verify that this is supported server side.